sssd_test_framework/roles/client.py

```diff
@@ -80,6 +81,11 @@ def __init__(self, *args, **kwargs) -> None:
         The D-bus destination for infopipe.
         """
 
+        self.smart_card: SmartCardUtils = SmartCardUtils(self.host)
```

Comment:
To add svc to the class (from a comment below), you'll need to add self.svc to the SmartCardUtils() call.